index.js

```diff
@@ -0,0 +1,17 @@
+// temp data, will be added to local storage
+const watchedVideos = []
+let watchCount = 0
+
+const watchCountEl = document.getElementById("watch-count-el")
+
+document.addEventListener("click", function(e){
```

Comment:
is it possible we add an event listener to just the button? If we have it on the whole document, this will run every time someone clicks the page, which could end up slowing down the site.